environment/src/main/java/com/sequenceiq/environment/environment/validation/cloudstorage/CloudStorageLocationValidator.java

```diff
@@ -34,6 +35,10 @@ public CloudStorageLocationValidator(CloudProviderServicesV4Endopint cloudProvid
 
     public void validate(String storageLocation, Environment environment, ValidationResultBuilder resultBuilder) {
         Optional<FileSystemType> fileSystemType = getFileSystemType(environment);
+        if (fileSystemType.isPresent() && "gs".equals(fileSystemType.get().getProtocol()) && !storageLocation.startsWith("gs://")) {
```

Comment:
use Filesystemtype enum here

Reply:
Thanks for the suggestions. I made the change and retested with both CB-CLI and UI.